packages/react-api/src/hooks/useTileFeatures.js

```diff
@@ -111,5 +118,10 @@ export default function useTileFeatures({
     [stopAnyCompute]
   );
 
-  return [onViewportLoad, fetch];
+  const onDataLoad = useCallback(({ tiles: [tile] }) => {
+    const tilesFormat = new URLSearchParams(tile).get('format');
+    setTileFormat(tilesFormat || 'mvt');
```

Comment:
An enum here for TileFormat ? TileFormat.MVT

Reply:
Done